ionic-app/www/js/directives/sessionDirective.js

```diff
@@ -0,0 +1,9 @@
+app_directives.directive('dddSession', function () {
```

Comment:
I find the naming of this directive and the template it's using a bit misleading